api/app/responses/api.py

```diff
@@ -106,6 +106,12 @@ def post(self):
                 user = user_repository.get_by_id(user_id)
                 response = response_repository.get_by_id_and_user_id(response.id, user_id)
                 self.send_confirmation(user, response)
+                
+                event = event_repository.get_event_by_response_id(response.id)
+                if event.event_type == EventType.CONTINUOUS_JOURNAL:
+                    all_admin_event_roles = event_repository.get_event_admins(event_id=event.id)
+                    for admin_event_role in all_admin_event_roles:
+                        self.send_confirmation(admin_event_role.user, response, admin_user=True)
```

Comment:
Sorry I should have cleaned up my comments after changing my mind about how the event admins can be queried. This should now be similar to what you had before since we queried for and now get `AppUser` objects.
```python
event_admins = event_repository.get_event_admins(event_id=event.id)
for event_admin in event_admins:
    self.send_confirmation(event_admin, response, admin_user=True)
```